include/mesh.h

```diff
@@ -14,10 +14,11 @@
 #ifdef USE_MPI
 #include "mpi.h"
 #endif
-// TBB
-#include <tbb/parallel_for.h>
-#include <tbb/parallel_for_each.h>
-
+// OpenMP
+#ifdef _OPENMP
```

Comment:
Is this ifdef added because of clang? Do we need to activate if clang is used? I mean by `-fopenmp=libomp`
https://releases.llvm.org/3.7.0/tools/clang/docs/ReleaseNotes.html#openmp-support

Reply:
This is added not specifically for clang, we have FindOpenMP so it will add the appropriate flag. Clang needs to be compiled with OpenMP separately. These include guards are there if OpenMP is not found, irrespective of the compiler.